diff/keys_test.go

```diff
@@ -39,10 +40,14 @@ func TestDiff(t *testing.T) {
 	}
 
 	g1, err := ucfg.NewFrom(oneGraph, opts...)
-	assert.NoError(t, err)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
this pattern we have quite often. How about:

```
func mustNewFrom(in interface{}, opts ...ucfg.Option) *ucfg.Config {
  c, err := ucfg.NewFrom(in, opts...)
  if err != nil {
    t.Fatalf("Failed to create new config for: %v\nreason:%v", spew.Sdump(in), err)
  }
  return c
}
```

Reply:
Done, and pushed.